js/renovation/utils/get_option_value.ts

```diff
@@ -0,0 +1,12 @@
+export function getOptionValue(
+  option:
+  number |
+  string |
+  boolean |
+  null |
+  undefined |
+  (() => number | string | boolean | null | undefined),
+):
+  number | string | boolean | null | undefined {
+  return typeof option === 'function' ? option() : option;
```

Comment:
oh, I'm not sure it's good idea.
1. we don't `option` naming in renovation
2. this code doesn't take into account object as a parameter or function return value
3. we're going to get rid of function type in dimension props

Reply:
i'll get rid of it, because we desided to get rid of functions in options